packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -221,6 +233,7 @@ class FlutterSkiaGoldFileComparator extends FlutterGoldenFileComparator {
       defaultComparator,
       platform,
       suffix: '${math.Random().nextInt(10000)}',
+      onCirrus: platform.environment.containsKey('CIRRUS_CI'),
```

Comment:
This should work but we could run into similar problems on LUCI if we're caching anything there - we're not right now but we might in the future.

Should we just make it opt in to put it in the pkg folder so that local users can do that?

Reply:
Oh this will, be different when I refactor for Luci. Do you mean we should default to using systemTempDirectory and set the flag for the other way around, to use the pkg folder if we are local?

Reply:
Yeah, although if we have to change this anyway for LUCI it's probably not a big deal either way.